routes/api.php

```diff
@@ -9,8 +9,11 @@
  * @see \Rogue\Providers\RouteServiceProvider
  */
 
+// Assets
+$router->get('images/{post}', 'ImagesController@show');
```

Comment:
I had originally considered moving this into a separate `routes/assets.php` group that was stateless & unprefixed (see 2a27ba6) but I think that's probably overkill. Instead, I'm just removing the `api` prefix on this file & applying it for each version below. Let me know what y'all think.

Reply:
That makes sense to me!